src/components/addCard/index.js

```diff
@@ -79,7 +81,9 @@ export default class AddCard extends Component {
   }
 
   componentDidMount() {
-    this.refs.cardNumberInput.focus()
+    if (this.props.autoFocus) {
+      this.refs.cardNumberInput.focus()
```

Comment:
Now I see the code I see why it might be broken. 

why aren't we setting `autoFocus` on the `<TextInput>`?
https://facebook.github.io/react-native/docs/textinput.html#props

If there is a good reason, maybe we can `.focus()` and do it on a very short `_.delay` as well 

This depends on how easy it is to reproduce and verify the fix though. If we can be confident we've fixed it 99.9% of the time, then I think we should try these alternatives.